elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -36,10 +41,13 @@ async def verify_attached_bugs_cli(runtime: Runtime, verify_bug_status: bool, ad
     if not advisories:
         red_print("No advisories specified on command line or in group.yml")
         exit(1)
-    validator = BugValidator(runtime, output="text")
+    validator = BugValidator(runtime, use_jira, output="text")
     try:
         await validator.errata_api.login()
-        advisory_bugs = await validator.get_attached_bugs(advisories)
+        if use_jira:
```

Comment:
Since we are passing use_jira to initialize `validator`, we can store that and check internally if this is true or not.
so here could be a single call to `validator.get_attached_bugs`
and get_attached_bugs can be a wrapper which checks and calls separate methods
get_attached_bz_bugs and get_attached_jira_bugs